helm/kube-bench-chart/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-bench
+  namespace: default
```

Comment:
Are we certain about the namespace?

Reply:
Yes I believe kube-bench is intended to run in default namespace